src/utils/index.js

```diff
@@ -1,7 +1,42 @@
+import { DATE_FORMAT_LONG, formatDate, toDate } from './date'
 import { exactlengthHint, maxlengthHint, minlengthHint } from './lengthHint/lengthHint'
+import { hasPermission, hasAllPermissions, hasAnyPermissions } from './hasPermission'
+import { toggleFullscreen, bindFullScreenChange, unbindFullScreenChange, isActiveFullScreen } from './fullscreen'
+import changeUrlforPager from './changeUrlforPager'
+import debounce from './debounce'
+import deepMerge from './deepMerge'
+import formatBytes from './formatBytes'
+import getAnimationEventName from './getAnimationEventName'
+import getScrollTop from './getScrollTop'
+import hasOwnProp from './hasOwnProp'
+import sortAlphabetically from './sortAlphabetically'
+import throttle from './throttle'
+import prefixClass from './prefixClass'
```

Comment:
nitpick: sort alphabetically
Also, it seems we now have `prefixClass` in both `utils` and `lib`. Is that intentional?

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/71/commits/4d668be5a23f87f743543366141a4330adf197ab

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/71/commits/7ee9f9c4637aaa04509d65ae2e40fcaabb5e6c6f